dio/lib/src/interceptors/imply_content_type.dart

```diff
@@ -5,6 +5,8 @@ import '../form_data.dart';
 import '../headers.dart';
 import '../options.dart';
 
+const _kReleaseMode = bool.fromEnvironment('dart.vm.product');
```

Comment:
Can you add the PR description as a comment here? That way nobody changes it back, hopefully, because they were warned.

Reply:
Of course, that makes sense.